array/DNAm/analysis/EWAS/summariseWithinCTDMPs.r

```diff
@@ -0,0 +1,272 @@
+par(mfrow = c(3,3))
+par(mar = c(4,4,1,1))
+par(mgp = c(2,0.75,0))
+for(i in 1:3){
+    qq(res[[i]][,"NullModel_SCZ_P"], main = cellTypes[i])
+}
+for(i in 1:3){
+    if("CCModel_SCZ_P" %in% colnames(res[[i]])){
+        qq(res[[i]][,"CCModel_SCZ_P"], main = cellTypes[i])
+    } else {
+        plot(0, xaxt = 'n', yaxt = 'n', bty = 'n', pch = '', ylab = '', xlab = '')
+    }
+}
+for(i in 1:3){
+    qq(res[[i]][,"FullModel_SCZ_P"], main = cellTypes[i])
+}
+
+dev.off()
+
+## Compare with and without CETYGO & cell composition
+png(file.path(resPath, "Plots","ScatterplotsWithinCTLRSCZEffect.png"), width = 800, height = 1200, res = 200)
+par(mfrow = c(3,2))
+par(mar = c(4,4,1,1))
+par(mgp = c(2,0.75,0))
+for(i in 1:3){
+    if("CCModel_SCZ_P" %in% colnames(res[[i]])){
+        plot(-log10(res[[i]][,"FullModel_SCZ_P"]), -log10(res[[i]][,"CCModel_SCZ_P"]), pch = 16, 
+        xlab = "With CETYGO", ylab = "Without CETYGO", main = cellTypes[i], col = "darkgrey")
+        abline(a = 0, b = 1)
+        plot(-log10(res[[i]][,"CCModel_SCZ_P"]), -log10(res[[i]][,"NullModel_SCZ_P"]), pch = 16, 
+        xlab = "With cell composition", ylab = "Without cell composition", main = cellTypes[i], col = "darkgrey")
+        abline(a = 0, b = 1)
+    } else {
+        plot(-log10(res[[i]][,"FullModel_SCZ_P"]), -log10(res[[i]][,"NullModel_SCZ_P"]), pch = 16, 
+        xlab = "With CETYGO", ylab = "Without CETYGO", main = cellTypes[i], col = "darkgrey")
+        abline(a = 0, b = 1)
+    }
+}
+dev.off()
+
+# Manhattan plots
+
+png(file.path(resPath, "Plots", "ManhattanPlotsWithinCTLM.png"), width = 1400, height = 1000, res = 200)
+par(mfrow = c(3,1))
+par(mar = c(4,4,0.5,0.5))
+par(mgp = c(2,0.75,0))
+for(i in 1:3){
+    res[[i]][,"chrm"][which(res[[i]][,"chrm"] == "X")]<-"23"
+    res[[i]][,"chrm"]<-as.numeric(as.character(res[[i]][,"chrm"]))
+
+    resMan<-data.frame("SNP" = rownames(res[[i]]), "P" = res[[i]][,"FullModel_SCZ_P"], "CHR" = res[[i]][,"chrm"],"BP" = res[[i]][,"start"])
+    resMan<-na.omit(resMan)
+    manhattan(resMan, genomewideline = -log10(9e-8), suggestiveline = -log10(1e-5), main = cellTypes[i])
+} 
+dev.off()
+
+
+#----------------------------------------------------------------------#
+# IDENTIFY DMPS
+#----------------------------------------------------------------------#
+
+## count DMPs at diff thresholds
+nDMPs<-matrix(ncol = 9, nrow = 4)
+colnames(nDMPs)<-outer(c("FullModel", "CCModel", "NullModel"),  cellTypes, paste, sep = ":")
+rowIndex<-1
+for(thres in c(9e-8, 1e-7, 1e-6, 1e-5)){
+    for(i in 1:3){
+        nDMPs[rowIndex,(i*3)-2]<-length(which(res[[i]][,"FullModel_SCZ_P"] < thres))
+        if("CCModel_SCZ_P" %in% colnames(res[[i]])){
+            nDMPs[rowIndex,(i*3)-1]<-length(which(res[[i]][,"CCModel_SCZ_P"] < thres))
+        }
+        nDMPs[rowIndex,(i*3)]<-length(which(res[[i]][,"NullModel_SCZ_P"] < thres))
+    }
+    rowIndex<-rowIndex+1
+}
+cbind()
```

Comment:
No arguments have been supplied to `cbind()` here. Is this unfinished code or a typo? Currently this should just return `NULL` right?